PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -3069,21 +3060,23 @@ struct HfTrackIndexSkimCreatorCascades {
   PROCESS_SWITCH(HfTrackIndexSkimCreatorCascades, processNoCascades, "Do not skim HF -> V0 cascades", true);
 
   void processCascades(SelectedCollisions const& collisions,
-                       aod::V0Datas const& v0s,
+                       soa::Join<aod::V0Datas, aod::V0Covs> const& v0s,
```

Comment:
Does this add a new dependency?

Reply:
No, `V0Covs` is also produced in the lambdakzerobuilder.